hack/generate/gen-cli-doc.go

```diff
@@ -32,7 +33,19 @@ func main() {
 		log.Fatalf("Failed to get current directory: %v", err)
 	}
 
-	err = doc.GenMarkdownTree(root, currentDir+"/doc/cli")
+	docPath := filepath.Join(currentDir, "doc", "cli")
```

Comment:
@joelanford can you make a separate PR with this change?

Reply:
See https://github.com/operator-framework/operator-sdk/pull/2278. I'll remove this change from this PR.